CHANGELOG.md

```diff
@@ -8,6 +8,27 @@ Try to keep listed changes to a concise bulleted list of simple explanations of
 
 ## [Unreleased]
 
+
+## [1.5.6] - 2024-07-17
+
+[Full changelog](https://github.com/openfga/openfga/compare/v1.5.5...v1.5.6)
+
+### Added
```

Comment:
Why were these skipped, can we add them?

- https://github.com/openfga/openfga/pull/1754
- https://github.com/openfga/openfga/pull/1742 - we should call out that this is a breaking change for people writing their own storage adapters

Reply:
> Why were these skipped
Because whoever creates the release is tasked with making a judgement call about what should and shouldn't be in a release. This person often is unfamiliar with most of the changes that were made since the last release. It's a toilous task that leads to imperfect results. 

Re: #1754 isn't impactful enough to our end users IMO. 

Re: #1742, I can add that but based on [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/#summary), this probably shouldn't have been a "chore" but rather a "breaking change".